CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Information on how to properly use the Singularity cache with Cromwell is now
 provided in the [Cromwell Singularity documentation](
 https://cromwell.readthedocs.io/en/stable/tutorials/Containers/#singularity).
 
+### Bug fixes
+
+* Introduced a JVM DNS TTL of three minutes, overriding the default of infinity. This setting is not user-configurable. 
```

Comment:
TOL any reason not to make it configurable?

Reply:
Choosing a TTL is a balance between receiving updates quickly and generating too much load on DNS servers.

Some applications naturally make a ton of lookups, like a web crawler for example. I think Cromwell is the opposite - we talk to Google, and a few Docker repos.

I don't suspect anyone will really want to change this so I'm taking a YAGNI approach (with PRs accepted).